packages/components/docs/sass.md

```diff
@@ -21499,7 +21501,7 @@ Slider styles
 
   .#{$prefix}--slider__range-label {
     @include type-style('code-02');
-    font-family: carbon--font-family('mono');
+    // font-family: carbon--font-family('mono');
```

Comment:
Any reason why this is being generated? 🤔

Reply:
probably ran the build script before I removed the comment. the bot should fix it but I can rebuild the sassdocs